# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/vm/SharedArrayObject.cpp
# Commit: df0c1ee4d0b0
# Full Hash: df0c1ee4d0b0f11ea3149daf4b2a7d2813de6fea
# Author: Steve Fink <sfink@mozilla.com>
# Date: 2021-09-09 15:22:41
# Regressor Bug: 1720422
# File Overlap Count: 2
# Description:
#   Bug 1720422 - Fill in the rest of the ArrayBuffer+ArrayBufferView hierarchy, and base more ArrayBufferViewObject public APIs off of new API r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D119857
# ==============================================================================

diff -r e0d16e395a5f -r df0c1ee4d0b0 js/src/vm/SharedArrayObject.cpp
--- a/js/src/vm/SharedArrayObject.cpp	Wed Sep 08 23:46:58 2021 +0000
+++ b/js/src/vm/SharedArrayObject.cpp	Wed Sep 08 23:46:58 2021 +0000
@@ -502,8 +502,3 @@
 JS_PUBLIC_API bool JS::ContainsSharedArrayBuffer(JSContext* cx) {
   return cx->runtime()->hasLiveSABs();
 }
-
-const JSClass* const JS::ArrayBuffer::UnsharedClass =
-    &ArrayBufferObject::class_;
-const JSClass* const JS::ArrayBuffer::SharedClass =
-    &SharedArrayBufferObject::class_;
